k8s/namespaces/family-public-law/fpl-case-service/prod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+spec:
+  values:
+    java:
+      ingressHost: fpl-case-service-prod.service.core-compute-prod.internal
+      environment:
+        RELEASE: NOW
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        MANAGE_CASE_UI_BASE_URL: https://manage-case.platform.hmcts.net
+        DOCMOSIS_TORNADO_URL: https://docmosis.platform.hmcts.net
+        GATEWAY_URL: https://gateway.ccd.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        SURVEY_URL_CASESUBMISSION: https://www.smartsurvey.co.uk/s/FamilyPublicLaw?pageurl=application_submission
+        FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD: GeneralAppWithoutNotice
```

Comment:
hmm, not related to this PR but it was meant to be temporary, needs to align envs

Reply:
as agreed, let's get the new structure in place first and then revisit environment specific parameters as part of tech debt